lib/scrub/session.ex

```diff
@@ -362,6 +362,10 @@ defmodule Scrub.Session do
     end
   end
 
+  # no metadata was recieved. Connection is either bad or PLC is in FAULT state
+  defp fetch_structure_templates(%{tag_metadata: []}),
```

Comment:
This is a tad petty and definitely non-blocking, but I want to help people build the habit of only using the `def fun(), do:` function syntax if it will all actually fit on one line. As opposed to `def fun() do`.

Reply:
Oh I agree. It looks like mix format thought my one liner was too long...